bento/build.gradle

```diff
@@ -7,10 +7,10 @@ ext.projectDescription = 'A framework for building modularized Android user inte
 apply from: "$rootDir/publishing.gradle"
 
 android {
-    compileSdkVersion Versions.COMPILE_SDK
```

Comment:
Q: compileSdkVersion is usually put outside of "defaultConfig"? any particular reason we put it inside defaultConfig?
https://developer.android.com/about/versions/12/setup-sdk?authuser=2

Reply:
Great catch! I don't know why I moved that.